consensus/src/test_utils/mock_state_computer.rs

```diff
@@ -101,7 +100,7 @@ impl StateComputer for MockStateComputer {
             .commit_to_storage(commit.ledger_info().clone());
         self.commit_callback
             .unbounded_send(commit)
-            .expect("Fail to notify about sync");
+            .ok_or_else(|| anyhow!("Failed to notify about sync"))?;
```

Comment:
nit, this is a test utils, probably expect is more desirable?

Reply:
:+1: